modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -0,0 +1,7 @@
+[id="creating-a-new-client-in-keycloak_{context}"]
+= Creating a new client in Keycloak
+
+.Procedure
+
+. Configure Keycloak.
```

Comment:
For one step procedures, use a bullet point, not a number. See Mod. Doc. guidelines for procedures (I have to reference this all the time!) https://redhat-documentation.github.io/modular-docs/#creating-procedure-modules 

Also, usually in procedures, if there's a sub-step, it's providing more information, an alternate way of completing the action, or giving a note or heads up. What do you think about turning these actions into one line, or one action? Something like this:

Configure Keycloak by editing the Argo CD and updating `.sso.spec.provider: keycloak`.